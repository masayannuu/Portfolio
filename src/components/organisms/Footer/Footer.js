import React from 'react'
import styled from 'styled-components'
import { Container } from 'reactbulma'

const StyledFooter = styled.footer`
  background-color: #a2c4c7;
`

const StyledDiv = styled.div`
  font-size: 0.8rem;
  color: #ffffff;
  font-feature-settings: 'palt';
  letter-spacing: 0.2em;
`

const Footer = () => (
  <StyledFooter className="footer">
    <Container>
      <StyledDiv className="content has-text-centered">
        <p>Thank you for reading!</p>
        <p> &copy; {new Date().getFullYear()} Masaya Morimoto</p>
      </StyledDiv>
    </Container>
  </StyledFooter>
)

export default Footer
