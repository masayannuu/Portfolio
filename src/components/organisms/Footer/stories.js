import React from 'react'
import { storiesOf } from '@storybook/react'

import Footer from './Footer'

storiesOf('Organims.Footer', module).add('Footer', () => <Footer />)
