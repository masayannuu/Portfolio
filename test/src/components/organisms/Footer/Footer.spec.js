import React from 'react'
import Enzyme from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Footer from '../../../../../src/components/organisms/Footer/Footer'

describe('Footer Footer Molecules', () => {
  it('should render Footer', () => {
    const wrapper = Enzyme.shallow(<Footer />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
