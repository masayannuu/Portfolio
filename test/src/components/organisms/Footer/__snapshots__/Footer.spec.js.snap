// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer Molecules should render Footer 1`] = `
<Footer__StyledFooter
  className="footer"
>
  <Container>
    <Footer__StyledDiv
      className="content has-text-centered"
    >
      <p>
        Thank you for reading!
      </p>
      <p>
         Â© 
        2018
         Masaya Morimoto
      </p>
    </Footer__StyledDiv>
  </Container>
</Footer__StyledFooter>
`;
